Testscripts/Windows/DM-HOTADD-STRESS-NG.ps1

```diff
@@ -91,7 +91,7 @@ function Main {
             }
             Write-LogInfo "Stress-ng is installed"
         }
-        Start-Sleep -Seconds 40
+        Start-Sleep -Seconds 120
```

Comment:
This is static waiting before starting looping. Should it be? Otherwise, what happens? Can we add it to while loop like $sleepPeriod = 240?

Reply:
During install stress-ng, the demand memory increase, sleep enough time, wait for the demand memory decrease and stable, add loop need a judgement to exit ahead of time, but it is hard to find this value, so I increase the sleep time.